src/IssueViz/Editor/ErrorTagging/ErrorTagTooltipProvider.cs

```diff
@@ -44,6 +45,7 @@ internal interface IErrorTagTooltipProvider
     [PartCreationPolicy(CreationPolicy.Shared)]
     internal class ErrorTagTooltipProvider : IErrorTagTooltipProvider
     {
+        private static readonly Uri dummyUri = new Uri("rules://sonarlint");
```

Comment:
Should we make it a real url just in case if we fail to override it due to a bug or something. Maybe "www.sonarsource.com" ? Just in case?

Reply:
I wondered about using a real URL but thought that might suggest to a casual reader that we expect to navigate to it.

Just setting the Uri property does nothing on its own - nothing will happen unless something registers to handle the event. So not having a real URL should be ok.